src/controllers/event.js

```diff
@@ -0,0 +1,32 @@
+const Events = require('../database/events')
+
+
+ exports.post = (req, res, next) => {
+
+    const newEvent = new Event({
```

Comment:
Make a function that works as a query to your database, it should take this large object of yours `req.body` and save it into the database, you can do it with both callback and promise, no restrictions.